test/vapor_test.exs

```diff
@@ -76,5 +76,27 @@ defmodule VaporTest do
       assert TestConfig.get!("bar", as: :string) == "manual bar"
       assert TestConfig.get!("baz", as: :string) == "file baz"
     end
+
+    test "reads config from toml" do
+      config =
+        Config.default()
+        |> Config.merge(Config.File.with_name("test/support/settings.toml"))
+
+      TestConfig.start_link(config)
+
+      assert(TestConfig.get!("foo", as: :string) == %{"foo" => "foo toml"})
```

Comment:
I would have thought this was `foo.foo` or `[“foo”, “foo”]`, but it looks like that’s to be handled in #14.

Reply:
Yea it feels odd to me as well but #14 can standardized this.